src/Util/Lock.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Friendica\Util;
+
+/**
+ * @file src/Util/Lock.php
+ * @brief Functions for preventing parallel execution of functions
+ *
+ */
+
+use Friendica\Core\Config;
+use Memcache;
+use dba;
+use dbm;
+
+/**
+ * @brief This class contain Functions for preventing parallel execution of functions
+ */
+class Lock {
+       /**
+         * @brief Check for memcache and open a connection if configured
+         *
+         * @return object|boolean The memcache object - or "false" if not successful
+         */
+        public static function memcache() {
```

Comment:
At this point it could be a `private static function` because we don't use this method outside of this file. Also the method name could be a little bit more descriptive (e.g. `openMemcache`)